internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -262,11 +300,13 @@ func (r *BundleReconciler) handleCompletedPod(ctx context.Context, u *updater.Up
 		return updateStatusUnpackFailing(u, fmt.Errorf("get bundle contents: %w", err))
 	}
 
+	// TODO: this doesn't apply to all bundle source types
```

Comment:
Nit: format this with a title-like thing on the first line so that the TODO issue creator makes a nicer GH issue for it when we merge?

Reply:
We actually have https://github.com/operator-framework/rukpak/issues/164 open for this so I decided to put a link to there instead